internal/server/config.go

```diff
@@ -621,6 +624,15 @@ func (s Server) loadConfig(config Config) error {
 			Resource: "infra",
 		})
 
+		org, err := data.GetOrganization(tx, data.ByName(config.OrganizationName))
```

Comment:
This seems like it will act weirdly if the config doesnt specify and organization name or domain. This should be in a check that the fields aren't empty strings.

Reply:
I kind of wanted to make it a required field. Might cause some complications for migrating from older versions. maybe I'll set it to a default value if it's missing